bin/extract_doc_versions.pl

```diff
@@ -144,20 +147,20 @@ ()
   print $outfile "<tr><th>language</th><th>Sum up to date</th><th>Sum translated</th></tr>\n";
 
   # number of english files to translate
-  my $sum_files=scalar keys %{$svninfo{"en"}};
+  my $sum_files=scalar keys %{$gitinfo{"en"}};
 
   # loop through languages
-  foreach my $lang (sort keys %svninfo) {
+  foreach my $lang (sort keys %gitinfo) {
     # loop through filenames
     my $up_to_date=0;
-    foreach my $dir_file (keys %{$svninfo{"en"}}) {
-      if (exists $svninfo{$lang}{$dir_file}) {
-        if ($svninfo{$lang}{$dir_file}{'version'} >= $svninfo{"en"}{$dir_file}{'version'}) {
+    foreach my $dir_file (keys %{$gitinfo{"en"}}) {
+      if (exists $gitinfo{$lang}{$dir_file}) {
+        if ($gitinfo{$lang}{$dir_file}{'version'} >= $gitinfo{"en"}{$dir_file}{'version'}) {
```

Comment:
The git version is a hash and there is no guarantee that the comparison will work.
The datetimes of the commits should be used instead.